app/views/auth/sessions/new.html.haml

```diff
@@ -1,30 +1,22 @@
 - content_for :page_title do
   = t('auth.login')
 
-- content_for :header_tags do
-  = render partial: 'shared/og'
+.simple_form.alternative-login
+  %h4= 'Redirecting...'
 
-- unless omniauth_only?
-  = simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
```

Comment:
Could we keep all this but add a new param instead?

Reply:
yes, I realized I can use the sso_redirect param that was defined in https://github.com/mastodon/mastodon/pull/26083, if I merge that one in

Reply:
Yay!

Reply:
Could actually merge this back upstream too with that.